### DISCUSSION TITLE
` ["<c-c><c-k>"] = "Abort"` doesn't work

### DISCUSSION THREAD

**Discussion by azzamsa** [NeogitOrg/neogit#1138] [Q&A] [answered]
Hi, 👋

How to debug why `["<c-c><c-k>"] = "Abort"` doesn't work on my end.
I have no way to abort my changes during interactive rebase.

```lua
    {
        "neogitorg/neogit",
        event = "VeryLazy",
        commit = "38dd297a905ec6869f4c20ea9184a3e514316e3b",
        config = function()
            local neogit = require("neogit")
            neogit.setup(})
        end,
    },
```

- When I press `<c-c><c-k>`

![2024-01-18_09-14_1](https://github.com/NeogitOrg/neogit/assets/17734314/6cb1d699-5214-4c07-b6d3-b858d95229bb)

- :verbose nmap `<c-c><c-k>`

![2024-01-18_09-14_2](https://github.com/NeogitOrg/neogit/assets/17734314/ef8a9d47-ec79-4872-9d0d-8f388f11e4ce)

- :verbose nmap `<c-c><c-c>` ,  :verbose nmap `<c-c><c-k>`

![2024-01-18_09-12](https://github.com/NeogitOrg/neogit/assets/17734314/9484b494-ea03-4afe-92c9-faf742a89cac)

```
NVIM v0.9.5
neovide 0.12.1
```

```
- **OS Name:**                                     Fedora Linux 39 (Workstation Edition)
- **OS Type:**                                     64-bit
- **GNOME Version:**                               45.3
- **Windowing System:**                            Wayland
```

**Comment by CKolkey** [comment]
You can see if it works if you bind it to something else - for example, I have `<c-k>` set to do something in my window manager, so I rebind abort like so:

```lua
      commit_editor = {
        ["<c-d>"] = "Abort",
        ["<c-c><c-k>"] = false,
      },
      rebase_editor = {
        ["<c-d>"] = "Abort",
        ["<c-c><c-k>"] = false,
      },
```

You'll want to determine if it's the function failing, or the key-mapping failing to invoke the function.

**Reply by azzamsa** [reply]
`["<c-c><c-g>"] = "Abort"` this works for me. I have no idea why `c-k` doesn't.

**Reply by CKolkey** [reply, marked as answer]
It sounds like something might be "eating" `c-k`. Not much I can do about that. Emacs chords aren't exactly "first class" in vim.

**Reply by azzamsa** [reply]
Thank you! :icecream:
